### PR TITLE
Brings in OpenSea API call to populate derivCards

### PR DESCRIPTION
**Pull request by notverybased**

### CODE REVIEW

**Review by DennisDv24 — APPROVED** [review]
looks good

File: package.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "@types/node": "^16.11.46",
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
+    "api": "^4.5.1",
```
Comment (Artegus) [inline package.json:13]: Why do we need this package?

Comment (DennisDv24) [reply package.json]: Fixed

**Review by Artegus — APPROVED** [review]
ok

**Pull request merged**